VCFv4.3.tex

```diff
@@ -33,15 +33,15 @@
 \newpage
 
 \section{The VCF specification}
-VCF is a text file format (most likely stored in a compressed manner). 
+VCF is a text file format (most likely stored in a compressed manner).
```

Comment:
If it isn't too much trouble, would you be able to revert the whitespace changes in this PR so only actual content changes show up in the diff?

Reply:
Just FYI (and I don't know why it isn't exposed better), if you append ?w=1 to the github URL, it will hide whitespace-only differences.

Reply:
The whitespace changes were automatically introduced by my text editor - I did not introduce them intentionally. I will revert them, and reintroduce in a different PR.

Reply:
Whitespace diffs should be fixed now.